ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
This path might need to be ``/opt/node_modules/{{ nodejs_app_name }}``. From [gpii/universal's README.md](https://github.com/GPII/universal/blob/master/README.md):

"please ensure that it is housed in a directory named node_modules otherwise its test cases will not be able to self-resolve"